hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -522,6 +522,11 @@ public final class OzoneConfigKeys {
   public static final String OZONE_MANAGER_FAIR_LOCK = "ozone.om.lock.fair";
   public static final boolean OZONE_MANAGER_FAIR_LOCK_DEFAULT = false;
 
+  public static final String OZONE_MANAGER_STRIPED_LOCK_SIZE_PREFIX =
+      "ozone.om.lock.striped.size.";
```

Comment:
When working on HDDS-8910 I found Ozone already uses `Striped` in a few places, and for `ContainerCache` the number of stripes is configurable:

https://github.com/apache/ozone/blob/98abce2e62396fcae554015d483d9d2c0fe0bb05/hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java#L164-L165

I think we should follow this naming convention for consistency, so I'd like to suggest a small change here:

```suggestion
      "ozone.om.lock.stripes.";
```